en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim.md

```diff
@@ -67,7 +67,7 @@ docker ps | grep choreo-connect-
 
      This will publish your first API on the Developer Portal as well as deploy it on Choreo Connect. You now have an OAuth 2.0 secured REST API that is ready to be consumed.
 
-### Step 4 - Subscribing to API and Get a Token
+### Step 4 - Subscribe to the API and Generate a Token
```

Comment:
"### Step 4 - Subscribe to the API and generate a token"

Reply:
Done!